### PR TITLE
Bugfix: Redirecting to `/login` instead of `/`

### LINKED ISSUE #1416: Bugfix: Need to login twice to get logged in
### What is the expected behaviour?
Should redirect  to the dashboard immediately after authorisation 

### What is the current behaviour?
If we get the password/email wrong the first time, need to press the login button twice to get logged in.

### How to reproduce the issue?

1. Provide a wrong email or password for login (for the first time)
2. Provide the correct creds and click the login button

https://user-images.githubusercontent.com/67645175/142008448-cd4cdca8-a24f-45f2-8253-ccb3715cc905.mov

### PR DESCRIPTION
**Pull request by arpitnath**
Fixes #1416 

After correct creds are provided and the user is authenticated, the user should not be redirected to '/login' instead should redirect to '/' 

**Loom** : https://www.loom.com/share/3a5c39b400d142e1b00cd6cec1d6f8bf

### CODE REVIEW

**Comment by arpitnath** [comment]
app: https://tooljet-pr-app.herokuapp.com/

**Review by Navaneeth-pk — DISMISSED** [review]

**Comment by Navaneeth-pk** [comment]
@arpitnath please resolve the conflicts.

File: frontend/src/LoginPage/LoginPage.jsx [hunk 1]
```diff
@@ -46,23 +46,23 @@ class LoginPage extends React.Component {
...
-  };
-
-  authFailureHandler = () => {
-    toast.error('Invalid email or password', {
-      toastId: 'toast-login-auth-error',
-      hideProgressBar: true,
-      position: 'top-center',
-    });
-    this.setState({ isLoading: false });
+    authenticationService.login(email, password).then(
```
Comment (sherfin94) [inline frontend/src/LoginPage/LoginPage.jsx:49]: We need `this.authSuccessHandler` and `this.authFailureHandler` as they are used by `GoogleSSOLoginButton` here: https://github.com/ToolJet/ToolJet/blob/709d3e3df4060f2ee193624e9f8477a442f92af2/frontend/src/LoginPage/LoginPage.jsx#L136-L139

Comment (sherfin94) [reply frontend/src/LoginPage/LoginPage.jsx:49]: This refactoring was done for this purpose. Can we make do it in such a way that this refactoring is kept intact?

Comment (sherfin94) [reply frontend/src/LoginPage/LoginPage.jsx:49]: This refactoring was only present in the `ee` repo, that is why this conflict occured.

**Review by sherfin94 — CHANGES_REQUESTED** [review]

File: frontend/src/LoginPage/LoginPage.jsx (hunk 1 above)
Comment (arpitnath) [reply frontend/src/LoginPage/LoginPage.jsx:49]: owh okay, sure 👍🏻

**Review by sherfin94 — APPROVED** [review]

**Pull request merged**